tincr/io/design/tincr_checkpoints.tcl

```diff
@@ -223,14 +223,15 @@ proc ::tincr::read_tcp {args} {
 	# this may work for importing designs...need to test using a design diff function...which TINCR has!
 	# need to test with a custom routed design that uses a different input pin for the bel route-through
 	# possible hack to complete nets that use a BEL route-through
-	foreach net [get_nets -filter {ROUTE_STATUS==ANTENNAS}] {
+	foreach net [get_nets $q -filter {ROUTE_STATUS==ANTENNAS}] {
 		route_design -nets -directive Quick $net -quiet
 	}
 	
-	
 	# unlock the design at the end of the import process...do we want to do this?
-	puts "Design importation complete. ($total_runtime seconds)"
-	remove_files -quiet *
+	if {!$quiet} {
+		puts "Design importation complete. ($total_runtime seconds)"
+	}
+	# remove_files -quiet *
```

Comment:
I added the -no_add option with the read_xdc commands. I uncommented this so all files are now removed. Keep in mind, that this file is still a work in progress, and so there are several places that I have commented out for testing different things, but don't want to delete.